### DISCUSSION TITLE
Account Suspension

### DISCUSSION THREAD

**Discussion by Kuntal-Gain** [community/community#119629] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I am writing to express my concern regarding the sudden suspension of my GitHub account (id : `KuntalGain` , email: `kuntalgain147@gmail.com`) . I am deeply troubled by this decision, as I have not violated any of the terms and conditions of your platform. My account suspension has left me in a difficult position, as my important works and projects are now inaccessible.

Additionally, I have not received any email from GitHub regarding the reason for the suspension, which leaves me unsure of the issue that led to this decision. This lack of communication has caused me further stress and inconvenience.

I kindly request an immediate review of this situation and prompt reinstatement of my account. My projects are vital to my work and community engagement, and this disruption is causing significant inconvenience.

Please let me know if any additional information or documentation is needed from my end to facilitate the review process. I appreciate your swift attention to this matter.

Thank you for your time and assistance.
Ticket id - `#2727936`

Best regards,
Kuntal Gain

**Comment by ettaboyle** [comment, marked as answer]
Hi @Kuntal-Gain-Flutter  

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
